custom_components/better_thermostat/events/trv.py

```diff
@@ -201,6 +204,13 @@ async def trigger_trv_change(self, event):
 
             _main_change = True
 
+        if self.real_trvs[entity_id]["advanced"].get("no_off_system_mode", False):
+            if _new_heating_setpoint == self.real_trvs[entity_id]["min_temp"]:
```

Comment:
Also, this is a bad comparison for floats - check equality / non-equality with `(not) math.isclose(a, b) `